hca/staging_area_validator.py

```diff
@@ -67,6 +72,9 @@ def __init__(
 
     @cached_property
     def validator(self):
+        """
+        validator function
+        """
```

Comment:
The docstring doesn't add much, and `validator` is a cached property, not a function.